src/mysqlsh_helpers.py

```diff
@@ -241,13 +311,16 @@ def _wait_until_mysql_connection(self) -> None:
         if not self.container.exists(MYSQLD_SOCK_FILE):
             raise MySQLServiceNotRunningError()
 
-    def _run_mysqlsh_script(self, script: str) -> None:
+    def _run_mysqlsh_script(self, script: str, verbose: int = 1) -> str:
         """Execute a MySQL shell script.
 
         Raises ExecError if the script gets a non-zero return code.
 
         Args:
             script: mysql-shell python script string
+            verbose: mysqlsh verbosity level
+        Returns:
+            stdout of the script
         """
         self.container.push(path=MYSQLSH_SCRIPT_FILE, source=script)
```

Comment:
there is a way to do this with pebble's `exec` method which is a lot cleaner. This was probably written before `exec` was implemented as a feature

Reply:
Ah that's weird when I read it before it looked like it was doing something else. Probably I didn't read it carefully enough 🙂